src/data/resolvers/motions.ts

```diff
@@ -83,6 +97,27 @@ export const motionsResolvers = ({
         (firstEvent, secondEvent) =>
           secondEvent.createdAt - firstEvent.createdAt,
       );
+      const blocktime = await getBlockTime(networkClient.provider, 'latest');
+
+      const timeToSubmitMS = motion.events[1].toNumber() * 1000;
```

Comment:
I usually do this as: 

```suggestion
      const timeToSubmitMS = `${motion.events[1]}000`;
```

But I'm good with whatever works :P